ui-tests/src/test/resources/features/customizations/connectors/api-client-connector-CRUD.feature

```diff
@@ -80,6 +81,14 @@ Feature: Customization - API Connector CRUD
       | api-connector-detail-config-host        | http://petstore.swagger.io-1 |
```

Comment:
Can you check if there's a reason for having steps `validate connector detail values by` and `validate values by element`.
Maybe we can ditch the connector one and use the universal one instead?

Reply:
Semantically, they are different pages.

The first one is for detail page in API Client Connectors while the second one is for "middle" step in connection creation process (where every connector can define different elements). Even more, the second one has desired text in the atribute `value` while the first one uses text outside of element definition.

However, since the step `validate connector detail values by "data-testid"` is used only here, we can remove it (whole `DetailSteps` class for API Client Connectors) and make some universal step. But I would leave it at it is in case we would like to add there some more steps in case to create some reproducers for API Client Connectors detail page.

Reply:
Let's not remove the step then, you are right that the API client connector detail page is subject to change (with authentication and hostnames)